From 1251b781273ce322b400579e1c3d18d4d4416802 Mon Sep 17 00:00:00 2001
From: Jonas van den Berg <jonas@vandenbe.com>
Date: Tue, 26 Mar 2024 19:34:29 +0100
Subject: [PATCH 2/5] Fix incorrect use of g_error g_error is only intended for
 programming errors that you cannot recover from. this on the other hand is a
 recoverable error.

---
 playerctl/playerctl-player.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/playerctl/playerctl-player.c b/playerctl/playerctl-player.c
index 013c8da..5d9f477 100644
--- a/playerctl/playerctl-player.c
+++ b/playerctl/playerctl-player.c
@@ -418,7 +418,7 @@ static void playerctl_player_get_property(GObject *object, guint property_id, GV
         GVariant *metadata = NULL;
         metadata = playerctl_player_get_metadata(self, &error);
         if (error != NULL) {
-            g_error("could not get metadata: %s", error->message);
+            g_printerr("could not get metadata: %s", error->message);
             g_clear_error(&error);
         }
         g_value_set_variant(value, metadata);
-- 
2.51.2

